test/Microsoft.Azure.ServiceBus.UnitTests/Microsoft.Azure.ServiceBus.UnitTests.csproj

```diff
@@ -6,6 +6,7 @@
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <LangVersion>7.3</LangVersion>
```

Comment:
Make it `latest`

Reply:
I set it explicitly to 7.3 because C# 7.3 is needed for the tests to compile after I added the second `MessageHandlerOptions` constructor. See [Unable to invoke overload when argument is func of non generic task and supplied as method](https://stackoverflow.com/questions/42642355/unable-to-invoke-overload-when-argument-is-func-of-non-generic-task-and-supplied#comment97274139_42642355) which describes the exact same situation where the compiler generates errors if LangVersion is smaller than 7.3:
```
OnSessionQueueTests.cs(63, 47): [CS0407] 'Task OnSessionQueueTests.ExceptionReceivedHandler(ExceptionReceivedEventArgs)' has the wrong return type
OnSessionQueueTests.cs(147, 47): [CS0407] 'Task OnSessionQueueTests.ExceptionReceivedHandler(ExceptionReceivedEventArgs)' has the wrong return type
OnSessionTopicSubscriptionTests.cs(104, 47): [CS0407] 'Task OnSessionTopicSubscriptionTests.ExceptionReceivedHandler(ExceptionReceivedEventArgs)' has the wrong return type
SenderReceiverClientTestBase.cs(262, 43): [CS0407] 'Task SenderReceiverClientTestBase.ExceptionReceivedHandler(ExceptionReceivedEventArgs)' has the wrong return type
SenderReceiverClientTestBase.cs(292, 43): [CS0407] 'Task SenderReceiverClientTestBase.ExceptionReceivedHandler(ExceptionReceivedEventArgs)' has the wrong return type
```

But I’m happy to bump it to `latest` if you think it’s better.

Reply:
Test project is free to use the latest and greatest. Rather than bumping it manually all the time, it could be on the latest all the time.